eor_limits/plot_eor_limits.py

```diff
@@ -789,15 +875,43 @@ def make_plot(
         help="List of papers to bold in caption.",
         default=None,
     )
+    parser.add_argument(
+        "--sensitivities",
+        type=str,
+        nargs="+",
+        default=None,
+        help=(
+            "List of names:::files for which to include sensitivies. "
+            "Files must be 21cmSense outputs."
```

Comment:
Maybe we should have more info about how to generate these?

Reply:
OK, added a quick reference to 21cmSense docs.